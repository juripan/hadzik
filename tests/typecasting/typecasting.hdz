bul x = pravda
cif n = cif(x)

sicke(cif i = 0, i < 26, i++){
    hutor(znak(cif('a') + i))
}

vychod(n)
