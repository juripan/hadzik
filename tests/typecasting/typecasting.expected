stdout: abcdefghijklmnopqrstuvwxyz| stderr: | returncode: 1